docs/architecture.md

```diff
@@ -1,280 +1,46 @@
 
-### Disk Chain
+## Flow DPS Live
 
-The [Disk Chain](https://pkg.go.dev/github.com/optakt/flow-dps/service/chain#Disk) uses the execution node's on-disk key-value store for the Flow protocol state to reconstruct the block sequence.
+The DPS Live binary fetches data from the Flow Network in two distinct ways:
 
-## Feeder
+* It acts as an [unstaked consensus follower](https://github.com/onflow/flow-go/blob/master/follower/consensus_follower.go#L20-L25), which allows it to have access to the protocol state and be notified when a new block is finalized.
+* It downloads block records from a Google Cloud Storage bucket which is continuously updated by the Flow Network.
 
-The Feeder component is responsible for streaming trie updates to the [Mapper](#mapping-state-machine).
-It outputs a state delta for each requested state commitment, so that the [Mapper](#mapping-state-machine) can follow the sequence of changes to the state trie and attribute each change to a block height.
+All of this information is then mapped into the DPS index, which is used by the DPS API.
 
-[Package documentation](https://pkg.go.dev/github.com/optakt/flow-dps/service/feeder)
+[![Flow DPS Live](./svg/flow_dps_live.svg)](./svg/flow_dps_live.svg)
 
-### Disk Feeder
+### Components
 
-The [Disk Feeder](https://pkg.go.dev/github.com/optakt/flow-dps/service/feeder#Disk) reads trie updates directly from an on-disk write-ahead log of the execution node.
+* [GCPStreamer](https://pkg.go.dev/github.com/optakt/flow-dps/service/cloud) -- Downloads block records from a Google Cloud Storage bucket.
+* [Consensus Tracker](https://pkg.go.dev/github.com/optakt/flow-dps/service/tracker#Consensus) -- Reads from the protocol database from an unstaked consensus follower and from the Execution Tracker.
+* [Execution Tracker](https://pkg.go.dev/github.com/optakt/flow-dps/service/tracker#Execution) -- Reads block records from the GCPStreamer.
+* [Mapper](https://pkg.go.dev/github.com/optakt/flow-dps/service/mapper) -- Uses the aforementioned components to build its index.
+* [Indexer](https://pkg.go.dev/github.com/optakt/flow-dps/service/index) -- Exposes a Reader and a Writer which give access to the index database.
+* [DPS API](https://pkg.go.dev/github.com/optakt/flow-dps/api/dps) -- Exposes the [DPS API](./dps-api.md), and reads from the DPS index.
 
-## Mapping State Machine
+## DPS APIs
 
-The Mapping FSM component is at the core of the DPS. It is responsible for mapping incoming state trie updates to blocks.
-In order to do that, it depends on the [Feeder](#feeder) and [Chain](#chain) components to get state trie updates and block information, as well as on the [Index](#index) component for indexing.
+The DPS API is a GRPC API that allows reading any data that was indexed by the Flow DPS, at any given height.
+It is the dependency of both the [Flow Rosetta API](https://github.com/optakt/flow-rosetta) and the [DPS Access API](https://github.com/optakt/dps-access-api).
```

Comment:
`The DPS API can also serve as the foundation for the Flow Rosetta API and the Flow Access API.`

Please note I changed the repository to `flow-access`. I think it's clear enough, and it will probably become the default way of providing the Access API for Flow anyway.